language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/BallerinaTextDocumentService.java

```diff
@@ -143,7 +143,7 @@ void setClientCapabilities(LSClientCapabilities clientCapabilities) {
                     position.getPosition());
             try {
                 return LangExtensionDelegator.instance()
-                        .completion(position, context, this.serverContext, cancelChecker);
+                        .completion(position, context, this.serverContext, null);
```

Comment:
This change is not required, right?

Reply:
I think we should find some other option to handle this case when debugging. There is a high possibility this temp change get committed. Worst case, a test case maybe :)

Reply:
@mohanvive as @IMS94 suggested, let's add a system property. I'm working on the fix and we can merge it along with this PR.

Reply:
Created #32825 to address that. May be we can close it with this PR.